tests/phpunit/DataValues/Time/IntlTimeFormatterTest.php

```diff
@@ -128,9 +128,8 @@ public function formatProvider() {
 			'2000/12/12 01:01:20.200000'
 		];
 
-		// Skip on HHVM to avoid .888500 vs. .888499 msec @see hhvm#6899
 		// https://bugs.php.net/bug.php?id=76822
-		if ( !defined( 'HHVM_VERSION' ) && version_compare( PHP_VERSION, '7.2', '<' ) ) {
+		if ( version_compare( PHP_VERSION, '7.2', '<' ) ) {
```

Comment:
We can drop this comparison too since we don't support PHP 7.2 anymore

Reply:
We'd have to drop these tests since they won't work on php 7.2+

Reply:
> they won't work on php 7.2+

:eyes:

Reply:
Thanks, done.